Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -10,18 +10,14 @@ public static class ArrayOperations
         public static void Sort(int[] array)
         {
 
-            var flag = true;
-            var temp = 0;
-            var numLength = 0;
-
             if (array == null)
             {
                 return;
             }
-            else
-            {
-                numLength = array.Length;
```

Comment:
I think I didn't express myself quite well. This line is not to be removed but it can stand without the else.

Reply:
done